src/features/ProtectTransaction/utils.ts

```diff
@@ -81,4 +82,13 @@ export abstract class ProtectTxUtils {
       return false;
     };
   }
+
+  public static getTypeFromNansenLabels(labels: string[]): NansenReportType {
```

Comment:
1. The name is slightly confusing especially in a TS context.
`getNansenReportType` or just `getReportType ?

2. Unit test please.